script.js

```diff
@@ -0,0 +1,75 @@
+function Book(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+let books = [];
+
+/* eslint-disable no-use-before-define */
+function removeBook(index) {
+  books.splice(index, 1);
+  storeBookLocally(books);
+  displayItems();
+}
+/* eslint-disable no-use-before-define */
+
+function displayItems() {
+  const items = document.querySelector('.book-item-wrapper');
+  items.replaceChildren();
+
+  const storedData = JSON.parse(localStorage.getItem('bookArray'));
+  if (storedData) {
+    books = storedData;
+    books.forEach((item, index) => {
+      const articleContainer = document.createElement('article');
+      const titleParagraph = document.createElement('p');
+      titleParagraph.innerHTML = item.title;
+      const authorParagraph = document.createElement('p');
+      authorParagraph.innerHTML = item.author;
+      const separator = document.createElement('hr');
+
+      const removeBtn = document.createElement('button');
+      removeBtn.classList.add('remove-btn');
+      removeBtn.innerHTML = 'Remove';
+
+      removeBtn.addEventListener('click', () => {
+        removeBook(index);
+      });
+
+      articleContainer.appendChild(titleParagraph);
+      articleContainer.appendChild(authorParagraph);
+      articleContainer.appendChild(removeBtn);
+      articleContainer.appendChild(separator);
+
+      items.appendChild(articleContainer);
+    });
+  }
+}
+
+function storeBookLocally(bookArray) {
+  window.localStorage.setItem('bookArray', JSON.stringify(bookArray));
+}
+
+function addBook(title, author) {
+  const newBook = new Book(title, author);
```

Comment:
- [OPTIONAL] I highly recommend you give every newly added unique identifier `id` which will help you in removing a book.